{
  "_annotation": {
    "cve_id": "CVE-2024-45848",
    "generated_from": "https://raw.githubusercontent.com/anchore/cve-data-enrichment/main/data/anchore/2024/CVE-2024-45848.json",
    "reason": "Added CPE configurations because not yet analyzed by NVD."
  },
  "cve": {
    "configurations": [
      {
        "nodes": [
          {
            "cpeMatch": [
              {
                "criteria": "cpe:2.3:a:mindsdb:mindsdb:*:*:*:*:*:python:*:*",
                "matchCriteriaId": "35FD9033-20FD-5F2C-9660-5891DE75E0A0",
                "versionEndExcluding": "24.7.4.1",
                "versionStartIncluding": "23.12.4.0",
                "vulnerable": true
              }
            ],
            "negate": false,
            "operator": "OR"
          }
        ]
      }
    ]
  }
}